lib/wprof/reporters/db_report.rb

```diff
@@ -0,0 +1,16 @@
+module Wprof
+  module Reporters
+    module DatabaseReport
+      def db_report
+        case @rec_type
```

Comment:
idem here https://github.com/Wolox/wor-prof/pull/1/files/109f71b2e7efda74442e5a5c2dc0198f9982e3ca#r266583040

Reply:
Sorry, but not. Case statements is better for code quality (it's so much clear to read) and its performance better than HASH and Send (Its true and it's verified!)

Reply:
> Case statements is better for code quality (it's so much clear to read)

That's relative, in my opinion it's easier to read the hash.
I would suggest to use hash so it's easy to maintain, instead of adding another case you could add another key value to the hash.

But in the end it's your choice and you are free to implement it as you want 😄

Reply:
Ok, i'm not agreed with that... for any skill level it's easy to understand a Select Case Statement. Hash its simply to read, but it's no clarify when your are using it for control flow with send. What is more readeable? maybe is a question of interpretation how you said.... For me its farly better than Hash+Send.